README.md

```diff
@@ -336,6 +337,18 @@ docker build -t $BASE/capella/readonly \
     readonly
 ```
 
+### 8. Docker image `t4c/client/backup`
+
+The T4C client backup image imports a model from a TeamForCapella server and exports it to a Git repository. It can be used as a backup solution, e.g. as cronjob every night.
```

Comment:
```suggestion
The T4C client backup image imports a model from a TeamForCapella server and exports it to a Git repository. It can be used as a backup solution, for example, from a scheduled job.
```